targets/unity-v2/make.js

```diff
@@ -484,7 +484,7 @@ function getCustomApiLogic(tabbing, apiCall)
 {
     if (apiCall.name === "ExecuteFunction")
         return tabbing + "var localApiServerString = PlayFabSettings.LocalApiServer;\n" +
-            tabbing + "if (!string.IsNullOrEmpty(localApiServerString))\n" +
+            tabbing + "if (!string.IsNullOrEmpty(localApiServerString) && PlayFab.Internal.PlayFabUtil.ShouldDebugFunction(request.FunctionName))\n" +
```

Comment:
Should this be in a separate PR for the Unity SDK?

Reply:
Already exists in Unity SDK PR I believe; it somehow bled into this. Removing on next commit.